Marlin/boards.h

```diff
@@ -53,6 +53,7 @@
 #define BOARD_GEN6_DELUXE       51   // Gen6 deluxe
 #define BOARD_SANGUINOLOLU_11   6    // Sanguinololu < 1.2
 #define BOARD_SANGUINOLOLU_12   62   // Sanguinololu 1.2 and above
+#define BOARD_ANET_10           621  // Anet v1.0
```

Comment:
Not sure about MELZI or SANGUINOLOLU, but the number is similiar to the near identical one.
so maybe 69 is also possible.